syntax = "proto2";

message ChangeCurrentSessionDayVentillationDurationMessage {
    required uint32 dayNumber = 2;
    required uint32 ventillationDuration = 3;
}
